service/resource/deployment/master_deployment.go

```diff
@@ -126,7 +126,7 @@ func newMasterDeployments(customObject kvmtpr.CustomObject) ([]*extensionsv1.Dep
 								Command: []string{
 									"/opt/k8s-endpoint-updater",
 									"update",
-									"--provider.bridge.name=" + key.NetworkBridgeName(key.ClusterID(customObject)),
+									"--provider.bridge.name=" + key.NetworkBridgeName(customObject),
```

Comment:
The usage of the key functions is now a bit easier.

Reply:
I like this 👍